README.md

```diff
@@ -8,4 +8,4 @@ More information: https://wonder-blocks.netlify.com/
 
 ## Contributing
 
-Please note – before contributing ensure that any design changes you are wanting to make are reflected in the [Wonder Blocks project in Zeplin](https://zpl.io/bl1owd1).
+Please note – before contributing ensure that any design changes you are wanting to make are reflected in the [Wonder Blocks project in Figma](https://www.figma.com/file/VbVu3h2BpBhH80niq101MHHE/Wonder-Blocks).
```

Comment:
Since we have designs in both places, perhaps you can make this more inclusive of that, like:

> Before contributing, ensure that any design changes you want to make are reflected in the latest designs. For recent changes, designs can be found in [Figma](link); otherwise, check [Zeplin](link).